libmamba/src/core/progress_bar_impl.cpp

```diff
@@ -1395,7 +1395,7 @@ namespace mamba
     {
         if (!m_is_spinner)
         {
-            auto seed = static_cast<std::size_t>(
+            auto seed = static_cast<unsigned int>(
```

Comment:
On the next line the implicit cast to `unsigned int` was happening, so it's better to be explicit here

Reply:
Actually, it's better to cast to `std::default_random_engine::result_type`, will do